docs/_articles/en/release-information.md

```diff
@@ -3,6 +3,38 @@ title: Release Information
 lang: en
 ---
 
+## [v2.4.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.4.0) (11-04-2020)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.3.0...v2.4.0)
+
+### Release Summary
+
+* RetireJS - Supports a new engine to detect vulnerable Javascript Library in the project
+* Underlying PMD Engine is upgraded to 6.29.0 (from 6.28.0)
+* Bug fixes 
+
+**Closed issues:**
+
+- Missing meta mime types detections [\#270](https://github.com/forcedotcom/sfdx-scanner/issues/270)
```

Comment:
Should we remove this since it was never released?

Reply:
ok, we can remove it.